mac/INSTALL.md

```diff
@@ -1,18 +1,27 @@
 Installation
 ============
 
-- Install Python3
+- Install **Python3**
```

Comment:
this is different to other INSTALL.md files - no need for bold here. The reason being is that I'd imagie most will jsut open in a text editor, not on github, etc.